docs/user-guide/rules/list.md

```diff
@@ -134,6 +134,10 @@ Within each cateogory, the rules are grouped by the [_thing_](http://apps.workfl
 
 - [`import-notation`](../../../lib/rules/import-notation/README.md): Specify string or URL notation for `@import` rules (Autofixable).
 
+### Keyframe selectors
```

Comment:
```suggestion
### Keyframe selector
```

I'll fix the title above and below to match in another pull request:
- Imports -> Import
- Keyframes - Keyframes name